tfx/components/trainer/executor_test.py

```diff
@@ -123,6 +133,7 @@ def _do(self, test_executor):
         exec_properties=self._exec_properties)
 
   def testGenericExecutor(self):
+    self._input_dict[constants.EXAMPLES_KEY] = self._single_artifact
```

Comment:
you don't need to set this as it's default to single artifact right?

Reply:
one thing i wasn't sure about was the ordering of tests (whether they're run in order from top to bottom) and if setUp is called before each test. single artifact should be the default, but for test where I'm not checking for multiple artifact behavior, I added it to be explicit. should that be reverted?

Reply:
resolved, was unclear on `setUp()` running before each test case (it does). will fix